### DISCUSSION TITLE
Shopify NFT app

### DISCUSSION THREAD

**Discussion by jiveyTO** [XRPLBounties/Proposals#52] [Ideas]
Create a Shopify app that will utilize the new NFT functionality (XLS-20). 

One idea is to mint and print an NFT that can be claimed via a QR code on a receipt, order confirmation page or order status page. This can be used for a contest like McDonald's monopoly game, where the user collects a set and wins a prize. Or collecting a certain number within one year gives them special VIP status.

Another idea is to create certificates of authenticity NFT that you would receive when purchasing an item like a bag.

Note: I worked at Shopify for 4 years and would be happy to help anyone tackling this. See this [page](https://help.shopify.com/en/manual/products/digital-service-product/nfts/blockchain-app-partners) for existing blockchain Shopify apps.

**Comment by jonathanlei** [comment]
That sounds like a great idea and a good use/demonstration of XLS-20 - could you create a proposal PR here? https://github.com/XRPLBounties/Proposals/tree/main/bounties feel free to reach out if you need any assistance.

**Comment by jiveyTO** [comment]
I am still working to submit this Shopify NFT bounty, but got a bit busy these past few weeks. 
Targeting next week to get something written up. 🤞🏽

**Comment by JST5000** [comment]
This bounty is now live for up to $17,000 in milestones! You can read more about it here: https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0052%20Shopify%20NFT%20App.md

**Comment by jvegalo** [comment]
@jiveyTO @JST5000 guys just to let you know, I started working on this bounty yesterday.

**Reply by jvegalo** [reply]
Also I am working alongside my co-worker @mthsaguiar

**Comment by ripci504** [comment]
Also have been working on this for a few weeks

**Comment by jiveyTO** [comment]
FYI, Shopify just released a token gated app example
https://github.com/Shopify/tokengating-example-app

**Comment by jiveyTO** [comment]
Also, Shopify just released some NFT educational videos for merchants

NFT intro video:
https://www.youtube.com/watch?v=PaeliqlOVVM
NFT strategy video:
https://www.youtube.com/watch?v=Dzy_o4Yzz3U

**Comment by happydream9028** [comment]
Hi, This is George.
Could you please explain more in detail about bounty?

**Reply by JST5000** [reply]
Hey George, not sure if you already saw this, but the bounty description is here: https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0052%20Shopify%20NFT%20App.md

Do you have a specific question beyond what's outlined there?

**Comment by jiveyTO** [comment]
📣 There is a double bounty opportunity here.  Shopify and Thirdweb are offering $9k to incorporate their APIs.
You'll likely be using Shopify's API already for this bounty, you'd just need to add Thirdweb into your solution.

https://replit.com/bounties/@thirdwebShopify/build-a-shopify-bloc

**Comment by JST5000** [comment]
Happy to announce that we've accepted @jvegalo's submission for Milestone 1 of the Shopify NFT App bounty! 

To confirm your GitHub account, @jvegalo please reply to this message with the GitHub account confirmation number (It will be provided via a follow up email).

**Reply by jvegalo** [reply]
Confirmation number: 6490

**Reply by JST5000** [reply]
Confirmed, thanks! :)
